helm/values-dev.yaml

```diff
@@ -78,15 +78,16 @@ microservice-chart:
     NODO_PER_PM_URI: /nodo/nodo-per-pm/v1/checkPosition
     NODO_READ_TIMEOUT: "10000"
     NODO_CONNECTION_TIMEOUT: "10000"
+    CARTS_MAX_ALLOWED_PAYMENT_NOTICES: "5"
```

Comment:
🚲 
```suggestion
    CARTS_MAX_ALLOWED_PAYMENT_NOTICES: 5
```

Reply:
Those properties are required as string because otherwise an error will be thrown at deploy time